app/src/main/java/com/kms/cura/view/fragment/RegDoc_ProfessionalFragment.java

```diff
@@ -29,41 +35,39 @@
     private StringListAdapter adapterDegree;
     private ImageButton btnNext, btnBack;
     private UpdateSpinner updateSpinner;
+    private View myFragmentView = null;
     private String spnText[] = {"Degree Types", "Areas of Speciality", "Facilities"};
 
+
     public RegDoc_ProfessionalFragment() {
 
     }
 
     @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+    }
+
+
+    @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        // Inflate the layout for this fragment
         View myFragmentView = inflater.inflate(R.layout.fragment_reg_doc__professional, container, false);
-        // get data from server
-        // ...
-        // init dummy data
-        degree = new ArrayList<>();
-        degree.add("A");
-        degree.add("B");
-        degree.add("C");
-
-        speciality = new ArrayList<>();
-        speciality.add("A");
-        speciality.add("B");
-        speciality.add("C");
-
-        facility = new ArrayList<>();
-        facility.add("A");
-        facility.add("B");
-        facility.add("C");
-        facility.add("D");
-        facility.add("E");
-        facility.add("F");
-        facility.add("G");
-        facility.add("H");
-        // ---------------------------------------
         updateSpinner = this;
+        if(DegreeModel.getInstace().degreeLoaded()){
```

Comment:
Don't need to check here because if loading failed will throws exception already